code/perception/src/traffic_light_detection/src/traffic_light_detection/traffic_light_training.py

```diff
@@ -6,12 +6,16 @@
 from torch.optim.lr_scheduler import ExponentialLR
 from torch.utils.data import DataLoader
 from torchvision.datasets import ImageFolder
-from data_generation.transforms import Normalize, ResizeAndPadToSquare, \
-    load_image
-from data_generation.weights_organizer import WeightsOrganizer
-from traffic_light_detection.classification_model import ClassificationModel
 from torchvision.transforms import ToTensor
-from traffic_light_config import TrafficLightConfig
+import sys
+import os
+sys.path.append(os.path.abspath(sys.path[0] + '/..'))
```

Comment:
This should not be required when you are installing the dependencies correctly via requirements.txt. Did you encounter an error in PyCharm or during execution in the built container?